utils/tests.py

```diff
@@ -224,3 +225,65 @@ def test_should_suggest_donation_probabilty_0(self):
             created=datetime.datetime.now())
         # Change downloads date again to be recent (however modal won't show because probability is 0.0)
         self.assertEqual(utils.downloads.should_suggest_donation(user, times_shown_in_last_day), False)
+
+    def test_clean_html(self):
+        ret = clean_html(u'a b c d')
```

Comment:
There are no tests for images... We do allow images right now... Neither for ul/li, ...

In particular the forum says:

> To format your text you can use the following HTML tags. a, img, strong, b, em, li, u, p, br, blockquote, code.

I know the original clean_html doesn't have tests for these but we should add these. Especially as we don't allow linking to http:// images, only to https:// images

Reply:
There is a test for ul, the thing is that we did not support it previously (so the text in the forum was wrong). Now I added other ul and img tests.

Note that the image tag with http or https source is allowed in this method, then is removed when we display it with a templatetag.